### ISSUE TITLE
The length of the columns should be between " "

### ISSUE THREAD

**Issue by GabrielCliseru** [mickwheelz/Salesforce-ERD-Tool#2]
postgresql;ELSA;Salesforce;"Account (Account)";"Account ID (Id)";1;"ID";18;"PRIMARY KEY";;
postgresql;ELSA;Salesforce;"Account (Account)";"Account ID (Id)";2;"ID";18"
postgresql;ELSA;Salesforce;"Account (Account)";"Deleted (IsDeleted)";3;"BOOLEAN";0"
postgresql;ELSA;Salesforce;"Account (Account)";"Master Record ID (MasterRecordId)";4;"REFERENCE";18;"FOREIGN KEY";"Salesforce";"Account (Account)";"Account ID (Id)"
postgresql;ELSA;Salesforce;"Account (Account)";"Account Name (Name)";5;"STRING";255"
postgresql;ELSA;Salesforce;"Account (Account)";"Last Name (LastName)";6;"STRING";80"

**Comment by mickwheelz** [comment]
Hi @GabrielCliseru , this tool has not been updated or supported in three years.

Please see http://schemapuker.com for the most recent version.

**Comment by GabrielCliseru** [comment]
We have used the https://schemapuker.herokuapp.com/ but i could not find an easy way to report these problems so the best I could do is to use this github repo. :)

**Comment by mickwheelz** [comment]
Hi @GabrielCliseru, thanks for the report. The best place is probably contacting me here: http://www.mickwheelz.net/contact/ or via twitter.com/mickwheelz_

If this issue relates to the current version of SchemaPuker I will take a look in to it :)

**Comment by GabrielCliseru** [comment]
Yes, they have been found today, on version BETA v0.3 . Keep it up, with a little bit of Regex we make it work and thanks for your contact details. If we find more we will let you know.
